k8s/configmap.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flink-config
+  labels:
+    app: flink
+data:
+  flink-conf.yaml: |+
+    parallelism.default: 2
+    jobmanager.memory.process.size: 4048m
+    taskmanager.memory.process.size: 4048m
+    taskmanager.numberOfTaskSlots: 1
+    taskmanager.state.local.root-dirs: /process-state/local-recovery
+    jobmanager.rpc.address: flink-jobmanager
+    taskmanager.rpc.port: 6122
+    jobmanager.rpc.port: 6123
+    blob.server.port: 6124
+    queryable-state.proxy.ports: 6125
+    restart-strategy: fixed-delay
+    restart-strategy.fixed-delay.attempts: 10
+    state.backend: rocksdb
+    state.backend.rocksdb.localdir: /process-state/rocksdb
```

Comment:
Let's uncomment this for the time being so that we don't use the PV in the normal case.

Reply:
I think master should reflect as close as possible to final result if we can. Let's comment it while testing for normal case.